style/app.scss

```diff
@@ -792,6 +779,136 @@ div.sidebar.collapsed
   background-color : #dedede;
 }
 
+.focus-menu-bar
+{
+  position: fixed;
+  bottom: 0;
+  width: calc(100% - 50px);
+  margin-left: 25px;
+  margin-right: 25px;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.focus-menu-toggle
+{
+  position: fixed;
+  left: 25px;
+  border-radius: 17px 17px 0 0;
+  border: none;
+  background-color: #a273d3;
+  color: white;
+  padding: 7px 15px 0 15px;
+  font-size: 20px;
+  width: 150px;
+}
+
+.focuses-list
+{
+  display: inline-block;
+  position: relative;
+  top: 3px;
+}
+
+.focus
+{
+  position: relative;
+  float: right;
+  margin: 5px -7px 0;
+  border-top-left-radius: 17px;
+  border-top-right-radius: 30px 53px;
+  background-color: #a172d2;
+  border: none;
+  box-shadow: 0px 12px 20px 6px rgb(0 0 0 / 50%);
+  padding: 3px 7px 0 7px;
+  width: 130px;
+  height: 45px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.focus button
+{
+  border: none;
+  background-color: transparent;
+  color: white;
+  font-size: 13px;
+  text-align: center;
+  width: 98px;
+  height: inherit;
+  cursor: pointer;
+}
+
+.focus:after{
+  content: "";
+  background-color: transparent;
+  height: 70px;
+  width: 40px;
+  border-radius: 50%;
+  border-width: 13px;
+  top: -5px;
+  border-style: solid;
+  position: absolute;
+  border-color: transparent transparent transparent #a172d2;
+  transform: rotate(315deg);
+  right: -32px;
+}
+
+.active-focus, .focus:hover
+{
+  z-index: 2;
+  border-top-left-radius: 30px 53px;
+  background-color: #bf7ce6;
+}
+
+.active-focus::before, .focus:hover::before
+{
+  content: "";
```

Comment:
@leenlab2 Sorry I forgot to add this comment. There are some *shared styles* between the `::before` and `::after` pseudoelements that you should pull out so that you don't need to repeat them. The CSS structure can look like:

```css
.active-focus::before,
.focus:hover::before,
.active-focus::after,.focus:hover::after
{
  ...
}

.active-focus::before,
.focus:hover::before
{
  ...
}

.active-focus::after,
.focus:hover::after
{
  ...
}
```